docs/examples/quickstart/test_quickstarts.rb

```diff
@@ -9,14 +9,25 @@
 CURL_ERROR = "curl: (7) Failed to connect to localhost port 5000: Connection refused\n"
 CURL_EMPTY = "curl: (52) Empty reply from server\n"
 
-quickstarts = [
-  { lang: 'go', setup: 'go build', server: './oso-go-quickstart' },
-  { lang: 'java', setup: 'make install', server: 'make run' },
-  { lang: 'nodejs', setup: 'npm i', server: 'npm run dev' },
-  { lang: 'python', setup: 'pip install --upgrade -r requirements.txt', server: 'FLASK_APP=app.server python -m flask run' },
-  { lang: 'ruby', setup: 'bundle', server: 'bundle exec ruby server.rb' },
-  # { lang: 'rust', setup: 'cargo build', server: 'cargo run' }
-]
+
+quickstarts = {
+  go: { setup: 'go build', server: './oso-go-quickstart' },
+  # TODO: add local_setup that tests against local Oso install for java
+  java: { setup: 'make install', server: 'make run' },
+  # TODO: add local_setup that tests against local Oso install for nodejs
+  nodejs: { setup: 'npm i', server: 'npm run dev' },
+  python: { setup: 'pip install --upgrade -r requirements.txt', server: 'FLASK_APP=app.server python -m flask run' },
+  ruby: { setup: 'bundle', server: 'bundle exec ruby server.rb' },
+  # rust: { setup: 'cargo build', server: 'cargo run' }
+}
+
+if ARGV.include? "local"
+  quickstarts[:go][:setup] = "go mod edit -replace github.com/osohq/go-oso@v0.20.1=../../../../languages/go && go build"
+  quickstarts[:python][:setup] = "pip install -e ../../../../languages/python/oso && pip install --upgrade -r requirements.txt"
```

Comment:
Is there any chance that the `--upgrade` flag will ignore the local `oso` we just installed via `-e` and fetch a remote copy?

Reply:
I tested it so don't think so.